stdlib/public/SwiftShims/RefCount.h

```diff
@@ -432,7 +431,14 @@ class RefCountBitsT {
   }
 
   LLVM_ATTRIBUTE_ALWAYS_INLINE
-  RefCountBitsT(RefCountBitsT<RefCountIsInline> newbits) {
+  RefCountBitsT(int dummy, RefCountBitsT<RefCountIsInline> newbits) {
```

Comment:
i found it hard to convert this into a static function as it is called by a constructor in a sub class (that doesn't have access to private state)

I went for passing a pointer instead of the actual value. Let me know what you all think